bootstrap.ts

```diff
@@ -10,6 +10,7 @@ $injector.require("errors", "./errors");
 $injector.requirePublic("fs", "./file-system");
 $injector.require("sysInfoBase", "./sys-info-base");
 $injector.require("hostInfo", "./host-info");
+$injector.require("osInfo", "./os-info");
```

Comment:
Ok, the interface will be IoS, which is totally strange and will be confused with iOS, so osInfo is ok